src/PostgREST/Config/Database.hs

```diff
@@ -9,31 +9,31 @@ import PostgREST.Config.PgVersion (PgVersion (..))
 
 import qualified Hasql.Decoders             as HD
 import qualified Hasql.Encoders             as HE
-import qualified Hasql.Pool                 as P
-import qualified Hasql.Session              as H
-import qualified Hasql.Statement            as H
-import qualified Hasql.Transaction          as HT
-import qualified Hasql.Transaction.Sessions as HT
+import qualified Hasql.Pool                 as SQL
+import           Hasql.Session              (Session, statement)
```

Comment:
Having `statement` and `SQL.statement` is really confusing. I mean - it was confusing before, too, with `H.statement` and `HT.statement`....

Can we do any better?

Reply:
Yes, that's not ideal either way. The proper solution is probably to set up a `PostgREST.SQL` module that wraps our various Hasql imports and disamboguates the API in this case. Then we could just do `import PostgREST.SQL as SQL` in all other modules that use Hasql, a bit like a explicit prelude. On the other hand, this adds a bit of indirection. WDYT?

Reply:
Hm. Do we actually need to use `Session.statement` here? We could also run the `queryPgVersion` in a transaction, right? This shouldn't affect performance, because it's only run once.

Reply:
Yes, good point - I'll make a proposal in a separate PR, merging this for now!